eosclubhouse/quests/hack2/inspector1.py

```diff
@@ -35,7 +35,34 @@ def step_begin(self):
         return self.step_launch
 
     def step_launch(self):
-        self.open_url_in_browser('https://www.hack-computer.com/about')
+        if self.is_cancelled():
+            return self.step_abort()
+
+        self.deploy_file('Inspector/base.css',
+                         '~/Documents/WebSources/InspectorQuest/', override=True)
```

Comment:
Instead of using `~/Documents` directly, we can think about using `GLib.get_user_special_dir(GLib.USER_DIRECTORY_DOCUMENTS)` so we can store the information in the real documents directory, that will work with locales, for example in my case is `~/Documentos` and this quest creates a new folder on my home directory instead of using that one.

Reply:
Fair call, I'll change it to use that.